src/main/java/org/springframework/retry/annotation/Retryable.java

```diff
@@ -36,6 +36,12 @@
 @Documented
 public @interface Retryable {
 
+	/**
+	 * Name of function to use for recover.
+	 * @return the name of retry function to use
+	 */
+	String recoverName() default "";
```

Comment:
I think `recover` is fully enough to indicate that we would like to have a strict method reference

Reply:
Agree.

Reply:
Well, but you still haven't renamed it just into a `recover()`.
And why do you call it `funciton` in the JavaDocs?
Isn't the story just about an annotated method in the same class where we have this `@Retryable`?
Why just don't explain that in this JavaDocs?

Also I think the `function` may confuse when we use a Spring Cloud Function. So, one may think this is a name of the function bean. Therefore I think we need to be more explicit and say that we want to have here a method name.

Reply:
Sure. Updated documentation for this field.